ocs_ci/ocs/ui/page_objects/storage_system_details.py

```diff
@@ -149,3 +149,21 @@ def is_storage_system_details_breadcrumb_present(self):
         if not is_present:
             logger.warning("Storage System Details breadcrumb is not present")
         return is_present
+
+    def get_estimated_days_from_consumption_trend(self):
```

Comment:
pls put this method into BlockAndFile class 
https://github.com/red-hat-storage/ocs-ci/blob/226d20858eb8db3e98a0e5850cc616632c385bb0/ocs_ci/ocs/ui/page_objects/block_and_file.py#L8

Reply:
done